Makefile

```diff
@@ -16,6 +18,9 @@ lint:
 build:
 	go build -o ${OUTPUT} -ldflags "${LDFLAGS_VERSION}" main.go
 
+core_api_gen:
+	oapi-codegen -include-tags=User,Organization,Invite -generate=types,client -package=astrocore "${CORE_OPENAPI_SPEC}" > ./astro-client-core/api.gen.go
```

Comment:
could the makefile test if `oapi-codegen` exists and install if it does not exist. Similar to the linter on line 14.

Reply:
should be doable, let me fix that